### ISSUE TITLE
conda_ops_dev_install.sh should only download dependencies 

### ISSUE THREAD

**Issue by sroet** [openpathsampling/openpathsampling#822]
One small thing that I noticed recently. 
The current conda_ops_dev_install.sh installs and then uninstalls OPS, which is a bit redundant. 

The file now looks like this
```
conda install -y -c omnia openpathsampling
conda update -y --all
conda uninstall -y openpathsampling
git clone https://github.com/openpathsampling/openpathsampling.git
cd openpathsampling && pip install -e . && cd ..
```
which could be converted to (using the `--only-deps` option of conda):
```
conda install -y -c omnia openpathsampling --only-deps
conda update -y --all
git clone https://github.com/openpathsampling/openpathsampling.git
cd openpathsampling && pip install -e . && cd ..
```

**Comment by dwhswenson** [comment]
Actually, the best thing is to do would be to have this and our Travis installation process be similar. Installing the last release means we get the dependencies of the last release; if dependencies have changed, there may be problems with the developer install (John pointed this out when I first introduced the dev install script: https://github.com/openpathsampling/openpathsampling/pull/728#issuecomment-350567482).

The current Travis installation was based on an original setup that was more complicated than needed, and then became a mess as parts of it were jettisoned in quick fixes. We used conda-build because MDTraj did; but we're pure Python, so don't need it. But that is why everything is in a conda recipe. Then conda-build started giving troubles, so we stopped using it. Then there were performance problems with conda, causing the install process to time out on Travis, so we had to find a workaround.

Currently we extract the requirements from the conda recipe, put them into a string on the bash command line, and request installs like that. The script that we use extract from the conda yaml is [`devtools/install_recipe_requirements.py`](https://github.com/openpathsampling/openpathsampling/blob/master/devtools/install_recipe_requirements.py), only the `--dry` option is being used for now. This process is ugly.

Long term, I think we want to keep all the requirements in a pip-style `requirements.txt`, even if pip can't install some things. Eventually, we want things that aren't pip-installable to be *optional* installs. The main problem with that is OpenMM, and you should be able to run OPS (with the toy engine) without installing OpenMM. But that will wait for 2.0, because of the way storage handles `simtk.unit`. Maybe even have two files: `requirements.txt` and `conda_requirements.txt`, to make it very clear. 

The pip-style files can be passed to conda as `conda install --file requirements.txt`. Then the dev install script would just clone the repo and run a conda command to install all the requirements. We'd also use a similar conda command to install on Travis.

**Comment by dwhswenson** [comment]
@sroet: This will be fixed by #840. Please take a look at the solution over there, and let me know if you have any thoughts on it!
